api/test/wfl/integration/modules/aou_test.clj

```diff
@@ -68,7 +69,16 @@
 
 ;; rr: GH-1071
 (deftest test-exec-on-same-workload-request
-  "executing a workload-request twice should not create a new workload"
-  (let [request (make-aou-workload-request)]
-    (is (= (workloads/execute-workload! request)
-          (workloads/execute-workload! request)))))
+  (testing
+    "executing a workload-request twice should not create a new workload"
+    (let [request (make-aou-workload-request)]
+      (is (= (workloads/execute-workload! request)
+             (workloads/execute-workload! request)))))
+  (testing
```

Comment:
I think a `deftest` is better actually as we can disable it if it fails (+ create a ticket to fix later)

Reply:
good idea, done